.travis.yml

```diff
@@ -1,9 +1,5 @@
 language: php
 
-env:
-  global:
-    - COMPOSER_FLAGS="--no-interaction"
```

Comment:
Travis already enables this by setting an environment variable for no interaction that composer sees.